webrender/Cargo.toml

```diff
@@ -45,6 +45,7 @@ plane-split = "0.13.3"
 png = { optional = true, version = "0.14" }
 rand ="0.4"
 rayon = "1"
+rendy-memory = { git = "https://github.com/zakorgy/rendy.git", branch = "memory_for_wr" }
```

Comment:
why are we using a fork?

Reply:
After https://github.com/szeged/webrender/pull/268 is merged, we can use the crates.io version